homeassistant/components/econet/sensor.py

```diff
@@ -22,29 +23,54 @@
 WIFI_SIGNAL = "wifi_signal"
 RUNNING_STATE = "running_state"
 
+SENSOR_NAMES_TO_ATTRIBUTES = {
+    TANK_HEALTH: "tank_health",
+    AVAILIBLE_HOT_WATER: "tank_hot_water_availability",
+    COMPRESSOR_HEALTH: "compressor_health",
+    OVERRIDE_STATUS: "override_status",
+    WATER_USAGE_TODAY: "todays_water_usage",
+    POWER_USAGE_TODAY: "todays_energy_usage",
+    ALERT_COUNT: "alert_count",
+    WIFI_SIGNAL: "wifi_signal",
+    RUNNING_STATE: "running_state",
+}
+
+SENSOR_NAMES_TO_UNIT_OF_MEASUREMENT = {
+    TANK_HEALTH: PERCENTAGE,
+    AVAILIBLE_HOT_WATER: PERCENTAGE,
+    COMPRESSOR_HEALTH: PERCENTAGE,
+    OVERRIDE_STATUS: None,
+    WATER_USAGE_TODAY: VOLUME_GALLONS,
+    POWER_USAGE_TODAY: None,  # Depends on unit type
+    ALERT_COUNT: None,
+    WIFI_SIGNAL: SIGNAL_STRENGTH_DECIBELS_MILLIWATT,
```

Comment:
The wifi signal sensor should use device class signal_strength.

https://developers.home-assistant.io/docs/core/entity/sensor#available-device-classes